boards/ssrc/icicle/nuttx-config/bootloader/defconfig

```diff
@@ -83,3 +83,4 @@ CONFIG_USBMSC=y
 CONFIG_USBMSC_COMPOSITE=y
 CONFIG_USBMSC_IFNOBASE=2
 CONFIG_USEC_PER_TICK=1000
+CONFIG_SIGNED_UBOOT=n
```

Comment:
CONFIG_ flags are basically meant for NuttX configuration. This only affects our own bootloader (application), so it is better to instad have something like "BOOTLOADER_VERIFY_UBOOT" in https://github.com/tiiuae/saluki-v1/blob/30c84f607f25f006a5c42bd00d3598603dd11572/src/hw_config.h#L108

( and / or same place for icicle bootloader configuration )

Reply:
The problem with CONFIG_ flags is also that in order to make it work, you'd need to add it into some Kconfig, that is, inside nuttx tree.